Adiciona documentação sobre decisões (#1)
cairo92/docs: README.md

Cairo 92
========

Este documento tem por objetivo registrar decisões tomadas em relação a administração do co-working Cairo 92. Cabe aos membros associados decidir sobre qualquer situação que não esteja aqui descrita.

## Despesas mensais

> O período no qual as despesas mensais são catalogadas e divididas entre os atuais membros presentes no co-working compreende o primeiro ao último dia do mês. Os pagamentos são realizados no mês seguinte ao mês catalogado.

Reviewer: ```suggestion
O período no qual as despesas mensais são catalogadas e divididas entre os atuais membros presentes no co-working compreende o primeiro ao último dia do mês.
Os pagamentos devem ser realizados no mês seguinte ao mês catalogado.
Pagamentos podem acontecer antes se houver pleno acordo entre as partes.
```

Author: Queria evitar frases soltas, agrupando as orações relacionadas em um mesmo parágrafo. Evitei o termo `devem` pelo entendimento que @bruno2ms, @lucastamoios e eu tivemos: enquanto não houver uma definição sobre como os pagamentos são cobrados, fica difícil definir um prazo. O que acontece quando, por exemplo, um pagamento não for efetuado no mês subsequente? Isto não foi decidido ainda.

O que você acha de alterarmos para o seguinte trecho?

```suggestion
O período no qual as despesas mensais são catalogadas e divididas entre os atuais membros presentes no co-working compreende o primeiro ao último dia do mês. Os pagamentos são realizados no mês seguinte ao mês catalogado, podendo acontecer antes se houver pleno acordo entre as partes.
```

> O pagador de uma determinada despesa deve ter em mente que, caso esteja com saldo positivo, apenas receberá o reembolso deste valor no mês seguinte à despesa. Caso o responsável não deseje ou possa arcar com a despesa, é sugerido que converse com os demais membros, de preferência os associados, para se encontrar uma forma de efetuar a compra ou pagamento pendente. Há ainda a possibilidade de se utilizar o caixa do co-working para pagamento de uma despesa, em casos decididos pelos associados.

Reviewer 2: ```suggestion
O pagador de uma determinada despesa deve ter em mente que, caso esteja com saldo positivo, apenas receberá o reembolso deste valor no mês seguinte à despesa. Caso o responsável não deseje ou não possa arcar com a despesa, é sugerido que converse com os demais membros, de preferência os associados, para se encontrar uma forma de efetuar a compra ou pagamento pendente. Há ainda a possibilidade de se utilizar o caixa do co-working para pagamento de uma despesa, em casos decididos pelos associados.
```

### Catálogo de despesas

> A atual ferramenta utilizada para catalogar as despesas é o [SplitWise][splitwise]. Todos os membros do co-working tem permissão de leitura e escrita, mas se faz necessário que alterações sejam realizadas por dois ou mais membros, a fim de diminuir a chance de ocorrerem erros no processo.

Reviewer: Sempre requerer 2 pessoas pra inserir itens pode ficar meio complicado. Se todo mundo instalar o aplicativo e receber as notificações, a gente pode sempre revisar o que for acontecendo e acredito que funcionaria bem.

Edições em itens já colocados realmente seria interessante haver uma comunicação mais explícita, talvez uma mensagem no slack com uma explicação.

Author: O que acontece é que já estamos tendo problemas por edições incorretas. A edição acompanhada minimiza o problema e ao mesmo tempo não dispensa a revisão posterior, realizada por quem vê as notificações.

Esta ideia de comunicar no Slack tem funcionado bem pra edições, como nós dois fizemos recentemente. Talvez deixar claro que não precisam ser duas pessoas reunidas pessoalmente, de forma que uma pré-comunicação no Slack resolva o problema?

Reviewer 3: Acredito que a mensagem no Slack, mesmo desacompanhado já aliviaria bem as coisas. Ou a integração entre SpliWise e Slack https://zapier.com/apps/splitwise/integrations/slack

Author: Sugeri os anúncios no `#tesouraria`. Vou trocar a redação para:

```suggestion
A atual ferramenta utilizada para catalogar as despesas é o [SplitWise][splitwise]. Todos os membros do co-working tem permissão de leitura e escrita, mas se faz necessário que alterações sejam previamente informadas no canal `#tesouraria` do Slack, a fim de diminuir a chance de ocorrerem erros no processo.
```

Cada despesa deve ser adicionada como uma entrada separada, na data em que tenha ocorrido.

> ## Itens permanentes

Reviewer: Rola linkar a planilha do inventário em algum lugar aqui.

Author: Não dá pra adicionar ela aqui por enquanto por estar com as seguintes permissões:

![image](https://user-images.githubusercontent.com/459906/75877765-2a86bd00-5df7-11ea-8779-c50a3d19aaa8.png)

Sou a favor de remover o acesso público, liberando acesso de  escrita para os associados e somente-leitura para os demais membros interessados em consultá-la. Precisamos decidir isto corretamente.

> Os itens permanentes do co-working, que não se enquadram como bens de consumo ou serviços catalogados como despesas mensais, são de propriedade dos associados. O valor dos mesmos é dividido entre as cotas possuídas pelos associados.

Reviewer: ```suggestion
Os itens permanentes do co-working que não se enquadram como bens de consumo ou serviços (geralmente referidos como despesas mensais), são de propriedade dos associados.
O valor dos mesmos, assim como os custos, são dividido entre as cotas possuídas pelos associados.
```

Author: Por que a adição de `geralmente referidos`? Utilizei o termo `despesas mensais` justamente por termos um tópico sobre isto no documento.

Qual a diferença de `valor` para `custos` no contexto?

O meu comentário sobre agrupar frases relacionadas em um mesmo parágrafo também é válido aqui.
